extensions/wikia/FounderEmails/FounderEmails.php

```diff
@@ -24,34 +24,30 @@
 /**
  * extension config
  */
-$wgFounderEmailsExtensionConfig = array(
```

Comment:
I updated the name of this variable to be less cumbersome (I don't think 'extension' or 'config' add any useful information) and more specific ('Events' describes what is actually held here).

Additionally, I removed the useless 'events' key since there's nothing but events in this structure.

Finally I removed the 'threshold' parameter since its always 1.  This allowed me to simplify logic that was checking whether we had some minimum number of events to "did we get any events".